src/rai_core/rai/communication/ros2/connectors.py

```
@@ -200,26 +209,100 @@ class ROS2HRIMessage(HRIMessage):
+                audios=audio_msgs,
+            )
+        )
+
 
 class ROS2HRIConnector(HRIConnector[ROS2HRIMessage]):
     def __init__(
         self,
         node_name: str = f"rai_ros2_hri_connector_{str(uuid.uuid4())[-12:]}",
-        targets: List[Tuple[str, TopicConfig]] = [],
-        sources: List[Tuple[str, TopicConfig]] = [],
+        targets: List[Union[str, Tuple[str, TopicConfig]]] = [],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)